README.md

```diff
@@ -181,6 +179,45 @@ class SendReachReportOperation {
 registerOperation("send-reach-report", SendReachReportOperation);
 ```
 
+### Frequency Capping
+
+If an an ad creative has been shown to the user too many times, fall back to a default option.
+
+In the ad-tech's iframe:
+
+```js
+// Fetches two ads in a list. The second is the proposed ad to display, and the first 
+// is the fallback in case the second has been shown to this user too many times.
+var ads = await adtech.GetAds();
+
+await window.sharedStorage.worklet.addModule("frequency_cap.js");
+var opaqueURL = await window.sharedStorage.runURLSelectionOperation(
+  "frequency-cap",
+  ads.urls,
+  {data: {campaignID: ads.campaignId}});
+document.getElementById("my-fenced-frame").src = opaqueURL;
+```
+
+In the worklet script (`frequency_cap.js`):
+
+```js
+class SelectURLOperation {
+  async function run(data, urls) {
+    // By default, return the default url (0th index). 
+    let result = 0;
+    
+    let count = await this.sharedStorage.get(data["campaign-id"]);
+    
+    // If under cap, return the desired ad.
+    if (count < 3) {
+      result = 1;
+      this.sharedStorage.set(data["campaign-id"], count + 1);
```

Comment:
Won't this do concatenation? Do you need to convert `count` to an integer here first?

Reply:
Cool, ok.  I wasn't sure, was just looking at https://www.tutorialspoint.com/how-to-add-a-number-and-a-string-in-javascript

Reply:
But LGTM otherwise.

Reply:
Wait, you're right! I could swear I tried this out in console. 

Good catch, updating.

Reply:
Done, PTAL